From: Valentine Fatiev <valentinef@nvidia.com>
Subject: [PATCH] BACKPORT:
 drivers/net/ethernet/mellanox/mlx5/core/en/tc_tun_gre.c

Change-Id: I61e04f475e71876acbd3a4fab72cf82f738dd95a
---
 drivers/net/ethernet/mellanox/mlx5/core/en/tc_tun_gre.c | 3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/net/ethernet/mellanox/mlx5/core/en/tc_tun_gre.c
+++ b/drivers/net/ethernet/mellanox/mlx5/core/en/tc_tun_gre.c
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: GPL-2.0 OR Linux-OpenIB
 /* Copyright (c) 2018 Mellanox Technologies. */
 
+#ifdef HAVE_TCF_TUNNEL_INFO
+
 #include <net/gre.h>
 #include "en/tc_tun.h"
 
@@ -96,3 +98,4 @@ struct mlx5e_tc_tunnel gre_tunnel = {
 	.parse_tunnel         = mlx5e_tc_tun_parse_gretap,
 	.encap_info_equal     = mlx5e_tc_tun_encap_info_equal_generic,
 };
+#endif /* HAVE_TCF_TUNNEL_INFO */
